spec/rubocop/cop/rspec/context_wording_spec.rb

```diff
@@ -214,4 +214,24 @@
       end
     end
   end
+
+  context 'when `AllowedPatterns:` is empty' do
```

Comment:
Why would anyone do that? What is not getting 100% coverage if we don’t add this spec?

Reply:
That method name tricked me too -- it turns out the allowed_patterns method is a combination of the Allowed Patterns and Prefixes. The earlier spec has a prefix.

Reply:
For this spec, I impoved the clarity by updating the Context name to include that both Prefixes and AllowedPatterns must be empty at the same time.

I don't think anyone should configure the cop this way (vs. simply turning it off in that case), but this does better describe the behavior of the Cop.